chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -40,8 +42,15 @@ pub struct PartialWitnessActor {
     /// Reed Solomon encoder for encoding state witness parts.
     /// We keep one wrapper for each length of chunk_validators to avoid re-creating the encoder.
     rs_map: RsMap,
+    /// Currently used to find the chain HEAD when validating partial witnesses,
+    /// but should be removed if we implement retrieving this info from the client
+    store: Store,
 }
 
+/// This is taken to be the same value as near_chunks::chunk_cache::MAX_HEIGHTS_AHEAD, and we
```

Comment:
can we directly use near_chunks::chunk_cache::MAX_HEIGHTS_AHEAD here?

Reply:
we could, but do we want to? I took the same value here, but decided against actually importing that constant, because it doesn't feel to me like the chunk logic and the partial witness logic should want to inherit some shared theoretical `MAX_HEIGHTS_AHEAD` value. In the future we might encounter reasons why for chunks a good value for this constant is 5, but for state witnesses it's actually 12. So for now I just redefined a new constant and kind of arbitrarily set it to the same value of 5, but let me know if you have thoughts